editor/editor_node.cpp

```diff
@@ -3068,7 +3068,7 @@ void EditorNode::register_editor_types() {
 	ClassDB::register_class<EditorSelection>();
 	ClassDB::register_class<EditorFileDialog>();
 	ClassDB::register_virtual_class<EditorSettings>();
-	ClassDB::register_class<EditorSpatialGizmo>();
+	ClassDB::register_class<SpatialEditorGizmo>();
```

Comment:
Why renaming this class ? All editor classes start with Editor, so it is easier to spot which classes are editor only.

Reply:
This PR merges SpatialEditorGizmo and EditorSpatialGizmo into a single class. I picked SpatialEditorGizmo as the name of the merged class but I don't mind changing it to the other.

Reply:
If it is an Editor only class, I think keeping EditorSpatialGizmo is better. But it is not that important ^^